### DISCUSSION TITLE
Implementing quantity discounts for products owned by a user

### DISCUSSION THREAD

**Discussion by 204050** [medusajs/medusa#2665] [Q&A]
The shop I am currently working on acts a bit like a marketplace as it is not really a standalone shop. I have understood that medusa currently does not support marketplaces, but can be extended for example with medusa-extender which I already use to integrate 'customers' from other systems into the store. The use case is that users who registered in the other system can order a test sample of their own product at a discounted price. The discount is affected by quantity. Regular customers should not be affected. When modifying the product's quantity in the cart the discount should also update accordingly.

I have tried several things, such as creating hidden variants only the owner can order, but it is error prone on top of duplicating all variants in the shop. I also tried overriding the price selection strategy or even the pricing service (with medusa-extender) but none seems to really work - for example in my case the quantity in the pricing context is always undefined.

I would be grateful if someone with more knowledge of the inner workings of medusa could point me into the right direction.

**Comment by mrfzd** [comment]
Hi,
Did you manage to get anywhere with this requirement?
